client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -177,6 +192,15 @@
      */
     ReaderSegmentDistribution getReaderSegmentDistribution();
 
+    /**
+     * The {@link StreamCut} provided to this API indicates that the stream has consumed data upto
+     * that point. Hence, it is safe for Pravega to truncate the stream using this {@link StreamCut} for this
+     * {@link ReaderGroup}.
+     *
+     * @param streamCuts Map of streams to their corresponding cuts with which truncation should be done.
```

Comment:
Could you elaborate?

Reply:
Rather than saying "truncation should be done" framing it as "retention is no longer required..." because there may be multiple reader groups or a minimum retention, so it won't actually result in truncation.